packages/renderer/src/lib/volume/VolumesList.svelte

```diff
@@ -185,6 +163,48 @@ async function fetchUsageData() {
 function gotoCreateVolume(): void {
   router.goto('/volumes/create');
 }
+
+let selectedItemsNumber: number;
+let table: Table;
+
+const columns: Column[] = [
+  new Column('Status', 'center', '70px'),
+  new Column('Name', 'left', '3fr'),
+  new Column('Age', 'left', '1fr'),
+  new Column('Size', 'right', '1fr'),
+];
+
+const tableHelper = new (class extends TableHelper {
+  selectable(volume: any): boolean {
+    return volume.inUse;
+  }
+
+  disabledText(): string {
+    return 'Volume is used by a container';
+  }
+
+  compare(column: string): ((object1: any, object2: any) => number) | undefined {
```

Comment:
should we have dedicated separate objects instead of one helper ?

like one comparator, one selectable

Reply:
Selection is per object/row so it needs to stay at the 'helper' level, but I've moved comparators to the columns.